### PR TITLE
Implementation of Resolve method 

### LINKED ISSUE #16: RPC Support For Resolve command
#### Problem
Need to support Resolve Command over RPC

 #### Proposed Solution
Expose the Resolve command, fetch the fabric ID from connect command

### PR DESCRIPTION
**Pull request by nirav1911**
Problem
Updated resolve method

Summary of Changes
Implemented Resolve method with RPC

Fixed #16

### CODE REVIEW

File: src/controller/python/chip-device-ctrl.py [hunk 1]
```diff
@@ -41,7 +41,8 @@
 from chip.ChipBleUtility import FAKE_CONN_OBJ_VALUE
 from chip.setup_payload import SetupPayload
 from xmlrpc.server import SimpleXMLRPCServer
-
+from enum import Enum
```
Comment (wahajsyed) [inline src/controller/python/chip-device-ctrl.py:44]: We should avoid deletions as much as possible for this Repo

File: src/controller/python/chip-device-ctrl.py [hunk 2]
```diff
@@ -639,6 +658,22 @@ def start_rpc_server():
...
+    if error is not None:
+        return { "status" : status.value, "error" :f'Unable to connect due to exception {error}' }
+    else:
+        if result is not None:
+            return { "status" : status.value, "result": result}
+        else:
+            return { "status" : status.value}
+
+def __check_supported_os()-> bool:
+    if platform.system() == 'Darwin':
```
Comment (wahajsyed) [inline src/controller/python/chip-device-ctrl.py:671]: ❤️

File: src/controller/python/chip-device-ctrl.py [hunk 3]
```diff
@@ -622,6 +625,21 @@ def echo_alive(message):
     print(message)
     return message
 
+def resolve(fabric_id: int, node_id: int) -> str:
```
Comment (wahajsyed) [inline src/controller/python/chip-device-ctrl.py:628]: return type should be a dict?

**Review by wahajsyed — COMMENTED** [review]
Added Comments

File: src/controller/python/chip-device-ctrl.py (hunk 1 above)
Comment (nirav1911) [reply src/controller/python/chip-device-ctrl.py:45]: Resolved

File: src/controller/python/chip-device-ctrl.py (hunk 3 above)
Comment (nirav1911) [reply src/controller/python/chip-device-ctrl.py]: Resolved

**Review by wahajsyed — APPROVED** [review]
LGTM

File: src/controller/python/chip-device-ctrl.py [hunk 4]
```diff
@@ -635,6 +636,17 @@ def echo_alive(message):
     print(message)
     return message
 
+def resolve(fabric_id: int, node_id: int) -> Dict[str, Any]:
```
Comment (cjandhyala) [inline src/controller/python/chip-device-ctrl.py:639]: why don't we just wrap around the actual do_resolve API, that way any updates made to these functions in future, we don't have to maintain our code

Comment (nirav1911) [reply src/controller/python/chip-device-ctrl.py:639]: Overall, we are not wrapping around the `do_`  methods as these methods don't return any values and just print the output for command line. In our case, we are looking for output and send it back to the client app. Also, these methods do some validation as command line users can pass any values but in our case, we are not doing any validation as we already have datatype defined.

**Review by Xiole-KKE — APPROVED** [review]
LGTM

**Review by mikaelhm — APPROVED** [review]

**Pull request closed without merging**